x/evm/artela/types/context.go

```diff
@@ -358,6 +411,7 @@ func (k *AspectRuntimeContext) SetAspectState(ctx *artelatypes.RunnerContext, ke
 		ctx.AspectId.Bytes(),
 		[]byte(key),
 	)
+	fmt.Println(fmt.Sprintf("SetAspectState set, ---aspectID:%s---, ---key:%s---, ---value:%s---", ctx.AspectId.String(), key, value))
```

Comment:
Better to remove the debug logs before merging

Reply:
@dumbeng This is for temporarily, the next step is to include state write/delete logs.

Reply:
Okay, keep it for now, we can do the final clean later.